vhdbuilder/packer/install-dependencies.sh

```diff
@@ -194,28 +194,49 @@ if [[ $OS == $UBUNTU_OS_NAME ]]; then
   done
 fi
 
+if [[ $OS == $UBUNTU_OS_NAME && $(isARM64) != 1 ]]; then  # no ARM64 SKU with GPU now
+  gpu_action="copy"
+  if grep -q "fullgpu" <<< "$FEATURE_FLAGS"; then
+    # force older driver on dedicated vhd to avoid build failures.
+    export NVIDIA_DRIVER_IMAGE_TAG="470.57.02-v0.0.2-rev20-sha-a0bbd5"
```

Comment:
So the dedicated VHD would now need the user to reinstall the drivers? Or it may not work for Nvidia V5 SKUs? I noticed a related comment in your email about this, and it seems like CSE is not dynamically updating the driver version for dedicated vhd.

Reply:
so right now this leaves the vhd at the 470 sku, but if compatible, it will used the 510 driver instead, which is most..

I should probably flip this since only ncv1 is compatible with 470 but not 510, but that happens to be what we run e2e with so I was worried about tests failing :D

maybe can just do that tho, and fix the tests

Reply:
Thanks for explaining, that makes sense. 

Flipping to 510 as default for dedicated VHD, would also probably reduce the average install latency for GPU dedicated VHD across SKUs.

Reply:
aaand I can't because we don't have quota to build on other vm sizes -_-

could install but not validate them..

Reply:
trying to get more gpu quota to switch off ncv1